lib/ship_compliant/version.rb

```diff
@@ -1,3 +1,3 @@
 module ShipCompliant
-  VERSION = "0.2.1"
```

Comment:
Yep. For reference if you've never seen it before: http://semver.org/ is a very commonly accepted convention